specifications/M1/verifiable.presentation.protocol.md

```diff
@@ -171,7 +171,40 @@ the `credentials` endpoint
 
 `POST /credentials`
 
-If successful, credentials will be created or updated and a `200 OK` returned.
+If the POST is successful, credentials will be created and a `201 Created` returned.
```

Comment:
```suggestion
If the POST is successful, credentials will be created and a HTTP `2XX` is returned.
```
If at some point in the future we want the API to return e.g. some ID of the created VC, or some status message, that would then mean we'd have to adapt the spec.